SalesforceSDK/Salesforce.SDK.App/Auth/PincodeDialog.xaml.cs

```diff
@@ -63,14 +63,20 @@ public PincodeDialog()
 
         protected override void OnNavigatedTo(NavigationEventArgs e)
         {
-            if (e.Parameter != null && !(e.Parameter is PincodeOptions))
+            // start with a default PincodeOptions object
+            Options = new PincodeOptions(PincodeOptions.PincodeScreen.Create, AccountManager.GetAccount(), "");
+            if (e.Parameter != null)
```

Comment:
The param is now a string (since we serialized it via Json) so I cannot check e.Param is PincodeOptions